### PR TITLE
Update author page

### PR DESCRIPTION
**Pull request by rdegges**
This PR adds phrasing to the author page that makes it clear that the page you're on contains a list of all posts by the given author. This has been brought up by a few different people over the last year, so this PR makes it a bit more obvious.

Below are some before/after pics of what this looks like (both for an author with and without a bio):

BEFORE

Screenshot from 2021-02-10 14-39-09

AFTER

Screenshot from 2021-02-10 14-38-50

BEFORE

Screenshot from 2021-02-10 14-39-56

AFTER

Screenshot from 2021-02-10 14-39-34

### CODE REVIEW

**Review by aaronpk — COMMENTED** [review]
Did you mean to include the OIE template stuff in here too?

**Comment by rdegges** [comment]
Whoops, no I did not. Will update that =)

File: _source/_layouts/author.html [hunk 1]
```diff
@@ -35,6 +35,9 @@ <h1 class="BlogPost-title">
 		    </div>
 		{% endif %}
 
+    {% assign first_name = page.full_name | split: " "[0]%}
```
Comment (mraible) [inline _source/_layouts/author.html:38]: Does this work on someone that has three words in their name?

Comment (aaronpk) [reply _source/_layouts/author.html:38]: It looks like that `first_name` isn't even used later

**Pull request closed without merging**